src/Compilers/CSharp/Portable/Symbols/TypeSymbol.SymbolAndDiagnostics.cs

```diff
@@ -4,6 +4,8 @@
 
 using System.Collections.Immutable;
 
+#pragma warning disable CS0660 // Warning is reported only for Full Solution Analysis
```

Comment:
Why not change the existing one to be a `SuppressMessageAttribute` on the type? Otherwise, you might run into the same issue if a new partial definition gets added in future.

Reply:
`SuppressMessageAttribute` doesn't work for compiler diagnostics.

Reply:
Ah yes, true.